src/components/vm/nics/nicAdd.tsx

```diff
@@ -235,27 +262,50 @@ export class AddNIC extends React.Component<AddNICProps, AddNICState> {
         const { idPrefix, vm } = this.props;
 
         const defaultBody = (
-            <Form onSubmit={e => e.preventDefault()} isHorizontal>
-                <NetworkTypeAndSourceRow idPrefix={idPrefix}
-                                         dialogValues={{ ...this.state, availableSources: this.props.availableSources }}
-                                         onValueChanged={this.onValueChanged}
-                                         connectionName={vm.connectionName} />
-
-                <NetworkModelRow idPrefix={idPrefix}
-                                 dialogValues={this.state}
-                                 onValueChanged={this.onValueChanged}
-                                 osTypeArch={vm.arch}
-                                 osTypeMachine={vm.emulatedMachine} />
-
-                <NetworkMacRow idPrefix={idPrefix}
-                               dialogValues={this.state}
-                               onValueChanged={this.onValueChanged} />
-
-                {domainIsRunning(vm.state) && vm.persistent &&
-                <PermanentChange idPrefix={idPrefix}
-                                 dialogValues={this.state}
-                                 onValueChanged={this.onValueChanged} />}
-            </Form>
+            <>
+                <Form onSubmit={e => e.preventDefault()} isHorizontal>
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-66f0c2f5-20251104-070631-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)